processor/transformer/transformer.go

```diff
@@ -256,6 +269,19 @@ func (trans *HandleT) Validate(clientEvents []TransformerEventT,
 	return trans.Transform(clientEvents, url, batchSize)
 }
 
+func (trans *HandleT) requestTime(s stats.Tags, d time.Duration) {
+	stats.NewTaggedStat("processor.transformer_request_time", stats.TimerType, s).SendTiming(d)
+}
+
+func statsTags(event TransformerEventT) stats.Tags {
+	return stats.Tags{
+		"dest_type": event.Destination.DestinationDefinition.Name,
+		"dest_name": event.Destination.Name,
```

Comment:
Do we really need a destination name tag when we have both type and ID

Reply:
I am thinking it could make debugging easier.

Reply:
It creates more problems than it solves, I have remove the `Destination.Name`

Reply:
Generally Influx would create more Indexes based on the tags so was just thinking of removing redundancy